projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/integration-card/integration-card-header.tsx

```diff
@@ -107,6 +112,14 @@ const IntegrationCardHeader = ( {
 						{ description && (
 							<span className="integration-card__description">{ description }</span>
 						) }
+						{ /* Show skeleton badge while loading status */ }
+						{ __isPartialData && (
+							<LoadingPlaceholder
```

Comment:
Would alternatively using the core `Animate` component on the Badge work? https://wordpress.github.io/gutenberg/?path=/story/components-animate--loading